cpg_infra/billing_aggregator/monthly_aggregate/main.py

```diff
@@ -68,25 +123,31 @@ async def process_and_upload_monthly_billing_report(invoice_month: str = None):
     data['cost'].fillna(0)
     data['key'] = data.topic + '-' + data.month + '-' + data.cost_category
     values: list = data.values.tolist()
-    append_values_to_google_sheet(OUTPUT_GOOGLE_SHEET, values)
+    updated = append_values_to_google_sheet(OUTPUT_GOOGLE_SHEET, values, invoice_month)
+
+    return f'{updated} cells appended for invoice month {invoice_month}', 200
 
 
 def abort_message(status: int, message: str):
     """Custom abort wrapper that allows for error messages to be passed through"""
     return abort(Response(json.dumps({'message': message}), status))
 
 
-def append_values_to_google_sheet(spreadsheet_id, _values):
+def append_values_to_google_sheet(spreadsheet_id, _values, invoice_month):
     """
     Creates the batch_update the user has access to.
     Load pre-authorized user credentials from the environment.
     TODO(developer) - See https://developers.google.com/identity
     for guides on implementing OAuth2 for the application.
     """
 
-    creds, _ = google.auth.default(
-        scopes=['https://www.googleapis.com/auth/spreadsheets']
-    )
+    year = invoice_month[:4]
```

Comment:
The hot trend is to be more assertive with Python where you expect the shape of specific data

```suggestion
    assert len(invoice_month) == 6
    year = invoice_month[:4]
```